test/acceptance/features/bindAppToService.feature

```diff
@@ -756,3 +756,36 @@ Feature: Bind an application to a service
             """
         Then Service Binding "binding-request-cross-ns-service" is ready
         And The application env var "BACKEND_HOST_CROSS_NS_SERVICE" has value "cross.ns.service.stable.example.com"
+
+    @negative
+    Scenario: Service binding with a missing app and configMap as a service
```

Comment:
We should focus on scenarios with app involved, in our case that could be generic generic test app. This particular case does not have much sense without the app, because a user could ask - what is the point of applying service binding against a config map, to get the secret containing the same content. Please rephrase the scenario stating what are the expectations, e.g. `Inject all configmap keys into application`